src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTable.java

```diff
@@ -2,91 +2,41 @@
 
 class LobbyGameTable extends JTable {
   private static final long serialVersionUID = 8632519876114231003L;
-  private GUID selectedGame;
-  private boolean inTableChange = false;
-  private final Font defaultFont = UIManager.getDefaults().getFont("Table.font");
-  private final Font italicFont = new Font(defaultFont.getFamily(), Font.ITALIC, defaultFont.getSize());
-
-  LobbyGameTable(final TableModel model) {
-    super(model);
-    getSelectionModel().addListSelectionListener(e -> {
-      if (!inTableChange) {
-        markSelection();
-      }
-    });
+  private static final Font defaultFont = UIManager.getDefaults().getFont("Table.font");
+  private static final Font italicFont = new Font(defaultFont.getFamily(), Font.ITALIC, defaultFont.getSize());
+
+  LobbyGameTable(final LobbyGameTableModel gameTableModel) {
+    super(gameTableModel);
+    final TableRowSorter<LobbyGameTableModel> tableSorter = new TableRowSorter<>(gameTableModel);
+    // by default, sort newest first
+    final int dateColumn = gameTableModel.getColumnIndex(LobbyGameTableModel.Column.Started);
+    tableSorter.setSortKeys(Collections.singletonList(new RowSorter.SortKey(dateColumn, SortOrder.DESCENDING)));
+    setRowSorter(tableSorter);
   }
 
   @Override
   public Component prepareRenderer(final TableCellRenderer renderer, final int rowIndex, final int colIndex) {
-    final Component c = super.prepareRenderer(renderer, rowIndex, colIndex);
-    if (this.dataModel instanceof TableSorter) {
-      final TableSorter tmodel = (TableSorter) this.dataModel;
-      if (tmodel.getTableModel() instanceof LobbyGameTableModel) {
-        final LobbyGameTableModel lmodel = (LobbyGameTableModel) tmodel.getTableModel();
-        final int row = tmodel.getUnderlyingModelRowAt(rowIndex);
-        final GameDescription gd = lmodel.get(row);
-        if (gd.getBotSupportEmail() != null && gd.getBotSupportEmail().length() > 0) {
-          c.setFont(italicFont);
-        } else {
-          c.setFont(defaultFont);
-        }
-      }
-    }
-    return c;
-  }
-
-  /**
-   * The sorting model will loose the currently selected row.
-   * So we need to restore the selection after it has updated
-   */
-  @Override
-  public void tableChanged(final TableModelEvent e) {
-    inTableChange = true;
-    try {
-      super.tableChanged(e);
-    } finally {
-      inTableChange = false;
-    }
-    restoreSelection();
-  }
-
-  /**
-   * record the id of the currently selected game.
-   */
-  private void markSelection() {
-    final int selected = getSelectedRow();
-    if (selected >= 0) {
-      selectedGame = (GUID) getModel().getValueAt(selected, LobbyGameTableModel.Column.GUID.ordinal());
+    final Component component = super.prepareRenderer(renderer, rowIndex, colIndex);
+    final LobbyGameTableModel lobbyGameTableModel = (LobbyGameTableModel) this.getModel();
+    final GameDescription gameDescription = lobbyGameTableModel.get(convertRowIndexToModel(rowIndex));
```

Comment:
`convertRowIndexToModel(rowIndex)` instead of just `rowIndex` is the core fix @ssoloff